docs/tutorial.md

```diff
@@ -622,9 +623,11 @@ The column format changes depending on the `type`:
     * `name` can be any string
     * `site` needs to be a HGVS-format protein variant specification, e.g.
      `p.Ala398Tyr`
-    * `function` should be either empty or a HGVS-format single-nucleotide
+    * `genomic_mutations` should be either empty or a HGVS-format single-nucleotide
```

Comment:
yep, but in all cases the genome build should be specified as `hg19` or `hg38`, not just `19` or `38` (here and in all mentions below)